packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -85,11 +85,34 @@ export const main = async (
 	// If no generators are found in positional params
 	// ask which generators should be run
 	if (!foundGenerators || !foundGenerators.length) {
-		const whichGenerators: QuestionCollection<{ generators: string[] }> = {
+		let generatorsOfCmsType: string[] = [];
+		const cmsType = isString(args.cmsType)
+			? args.cmsType.toLocaleLowerCase()
+			: null;
+
+		if (cmsType) {
+			if (['wp', 'drupal', 'any'].indexOf(cmsType) == -1) {
+				console.log(
+					chalk.yellow(`Invalid cmsType: ${cmsType}. Showing all generators.`),
+				);
+			} else {
+				generatorsOfCmsType = generators
+					.filter((generator) => {
+						return generator.cmsType === cmsType || generator.cmsType === 'any'
+							? true
+							: false;
+					})
```

Comment:
```suggestion
					.filter((generator) => generator.cmsType === cmsType || generator.cmsType === 'any'
					})
```

Since you split out the other logic for the error handling, you can use an implicit return here and get rid of the ternary. idc as much about the implicit return, but the ternary should definitely go here :)